config/locales/active_admin.de.yml

```diff
@@ -294,6 +294,7 @@ de:
         flash:
           article_online: "Dieser Artikel ist nun online"
           article_offline: "Dieser Artikel ist nun offline"
+          article_not_edited: "Artikel konnte nicht gefunden und verarbeitet werden."
```

Comment:
How can this happen? If it isn't findable then you shouldn't see it in the index, right?